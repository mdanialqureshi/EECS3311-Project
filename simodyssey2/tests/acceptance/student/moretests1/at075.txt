move(E)
land
move(SW)
move(W)
move(N)
