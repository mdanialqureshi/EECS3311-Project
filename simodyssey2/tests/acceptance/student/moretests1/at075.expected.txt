  state:0.0, ok
  Welcome! Try test(1,1,1,1,30)
->move(E)
  state:0.1, error
  Negative on that request:no mission in progress.
->land
  state:0.2, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.3, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.4, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.5, error
  Negative on that request:no mission in progress.
